docs/content/docs/tutorials/sagemaker-example.md

```diff
@@ -0,0 +1,362 @@
+          s3DataDistributionType: FullyReplicated
+      contentType: text/libsvm
+      compressionType: None
+    - channelName: validation
+      dataSource:
+        s3DataSource:
+          s3DataType: S3Prefix
+          # The input path of our validation data 
+          s3URI: s3://'$SAGEMAKER_BUCKET'/sagemaker/xgboost/validation
+          s3DataDistributionType: FullyReplicated
+      contentType: text/libsvm
+      compressionType: None
+' > ./training.yaml
+```
+
+Use your `training.yaml` file to create a SageMaker training job:
+
+```bash
+kubectl apply -f training.yaml
+```
+
+After applying your `training.yaml` file, you should see that your training job was successfully created:
+
+```bash
+trainingjob.sagemaker.services.k8s.aws/ack-xgboost-training-job-7420 created
+```
+
+You can watch the status of the training job with the following command:
+
+```bash
+kubectl get trainingjob.sagemaker --watch
+```
+
+It will a take a few minutes for `TRAININGJOBSTATUS` to be `Completed`.
+
+```bash
+NAME                            SECONDARYSTATUS   TRAININGJOBSTATUS
+ack-xgboost-training-job-7420   Starting          InProgress
+ack-xgboost-training-job-7420   Downloading       InProgress
+ack-xgboost-training-job-7420   Training          InProgress
+ack-xgboost-training-job-7420   Completed         Completed
+```
+
+If your training job failed, look to the `status.failureReason` after running the following command:
+```bash
+kubectl describe trainingjobs $JOB_NAME
+```
+
+## Next Steps 
+
+For more examples on how to use the SageMaker ACK service controller, see the [SageMaker controller samples repository][sagemaker-samples]. 
+
+### Cleanup
+
+You can delete your SageMaker training job with the `kubectl delete` command.
+```bash
+kubectl delete trainingjob $JOB_NAME
+```
+
+To remove the SageMaker ACK service controller, related CRDs, and namespaces see [ACK Cleanup][cleanup].
+
+It is recommended to delete any additional resources such as S3 buckets, IAM roles, and IAM policies when you no longer need them. You can delete these resources with the following commands or directly in the AWS console.
+
+```bash
+# Delete S3 bucket
+aws s3 rb s3://$SAGEMAKER_BUCKET --force
+
+# Delete SageMaker execution role
+aws iam detach-role-policy --role-name $SAGEMAKER_EXECUTION_ROLE_NAME --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
+aws iam detach-role-policy --role-name $SAGEMAKER_EXECUTION_ROLE_NAME --policy-arn arn:aws:iam::aws:policy/AmazonS3FullAccess
+aws iam delete-role --role-name $SAGEMAKER_EXECUTION_ROLE_NAME
+
+# Delete IAM role created for IRSA
+aws iam detach-role-policy --role-name $OIDC_ROLE_NAME --policy-arn arn:aws:iam::aws:policy/AmazonSageMakerFullAccess
+aws iam delete-role --role-name $OIDC_ROLE_NAME
+```
+
+To delete your EKS clusters, pods, or nodes, see [Amazon EKS Setup - Cleanup][cleanup-eks].  
+
+[configure-permissions]: https://aws-controllers-k8s.github.io/community/docs/user-docs/authorization/
```

Comment:
I've been using relative paths for the moment, in case we change root URLs in the future. 

Also the `/docs` is a Hugo setting, because we are viewing the "documentation" part of the website - as opposed to the API reference part. That is a fluid property that's open to discussion, but it's essentially because of our file structure.